### ISSUE TITLE
how use multi-classes logits in detection calibration?

### ISSUE THREAD

**Issue by VCBE123** [EFS-OpenSource/calibration-framework#34]
Since we use **inverse_sigmoid** to reconstruct the logits, Why not use the multi-class output directly? 
So the LogisticCalibration model can learn the correlation between different classes?

**Comment by fabiankueppers** [comment]
Hi @VCBE123 ,
including multiple classes is a little bit more complicated as it would require an input array of shape (N, K, D) with N samples, K classes and D feature dimensions (confidence per class, position, etc.). However, this does not follow any standards for logistic calibration so that we decided to leave this out.
Nevertheless, you can model correlations between different classes by simply appending the remaining class confidences to the feature dimension D, so that you can pass an array of shape (N, D + K) to obtain calibrated confidences but only for the k'th class.
Hope this helps.
